guides/common/modules/proc_enabling-openscap.adoc

```diff
@@ -16,6 +40,11 @@ To use the OpenSCAP plugin and content on external {SmartProxies}, you must enab
 --foreman-proxy-plugin-openscap-ansible-module true \
 --foreman-proxy-plugin-openscap-puppet-module true
 ----
+ifdef::katello,orcharhino[]
++
+If you want to use Ansible to deploy compliance policies, you must enable it first.
+For more information, see {ManagingConfigurationsAnsibleDocURL}[_{ManagingConfigurationsAnsibleDocTitle}_].
+endif::[]
```

Comment:
I'd move these Ansible/Puppet notes our of the _.Procedure_.

Are they prerequisites or not? If not, then they should go to _.Next steps_.

Like this, it's unclear when to enable them - before or after these modules?

Reply:
Moved to next steps. However, due to lack of test infra, I did not test enabling Ansible/Puppet content on Smart Proxies without foreman_ansible/foreman_puppet installed.